Utilities/Thread.cpp

```diff
@@ -22,10 +22,9 @@
 static void report_fatal_error(const std::string& msg)
 {
 #ifdef _WIN32
-	const auto& text = msg + "\n\nPlease report this error to the developers. Press (Ctrl+C) to copy this message.";
-	MessageBoxA(0, text.c_str(), "Fatal error", MB_ICONERROR); // TODO: unicode message
+	MessageBoxA(0, msg.c_str(), "Fatal error", MB_ICONERROR); // TODO: unicode message
 #else
-	std::printf("Fatal error: %s\nPlease report this error to the developers.\n", msg.c_str());
+	std::printf("%s", msg.c_str());
```

Comment:
And "Fatal error", and "\n" got removed... Do you ever look at the diff, what you are actually doing?